### PR TITLE
Document SHOW FULL TABLES

### PR DESCRIPTION
**Pull request by msudberg**

### CODE REVIEW

File: doc/user/content/overview/arangements.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+    weight: 4
+---
+
+We call the mechanisms that maintain materialized views for Materialize dataflows **arrangements**, and understanding them better can help you make decisions that will reduce memory usage while maintaining performance.
+
+<!-- trade off memory and performance -->
+
+## What are arrangements?
+
+Arrangements are streams of update triples composed of `(data, time, diff)`. `data` is structured as a key-value pair, and arrangements are indexed on the `data` keys. An arrangement records all updates, but an index only maintains the current accumulation of `diff` for each `(data, time)` pair.
```
Comment (awang) [inline doc/user/content/overview/arangements.md:16]: This is the first place where "index" is introduced, and I'm having trouble understanding what you're referring to.  Are you intending to compare this to the index used in a traditional database?

Related, wdyt of adding something like:
> Arrangements are the streaming dataflow equivalent of relational database indexes.

**Comment by awang** [comment]
There's a bunch of changes in this PR that are related to arrangements; I assume that wasn't intentional?

File: doc/user/content/sql/show-tables.md [hunk 2]
```diff
@@ -16,7 +16,8 @@ instances.
 Field | Use
 ------|-----
 _schema&lowbar;name_ | The schema to show tables from. Defaults to `public` in the current database. For available schemas, see [`SHOW SCHEMAS`](../show-schemas).
-`EXTENDED` | Returns system tables as well as user-created tables. By default, only user-created tables are returned.
+**EXTENDED** | Returns system tables as well as user-created tables. By default, only user-created tables are returned.
+**FULL**  | Returns table type (`system`, `user`, or `temp`).
```
Comment (awang) [inline doc/user/content/sql/show-tables.md:20]: Proposed change (I'm not sure if this is accurate though!) -- change text to:

> Returns a column `type` that contains the table type (`system`, `user`, or `temp`).

**Review by awang — CHANGES_REQUESTED** [review]
the SHOW FULL TABLES stuff generally lgtm, but I think you accidentally merged more than you intended?

File: doc/user/content/ops/diagnosing-using-sql.md [hunk 3]
```diff
@@ -8,8 +8,7 @@ menu:
 
 ## Diagnostic queries that can help understand apparent performance defects.
 
-Sections are meant to be thematically related based on what problems are
-experienced.
+Sections are meant to be thematically related based on what problems are experienced.
```
Comment (sploiselle) [inline doc/user/content/ops/diagnosing-using-sql.md:11]: Passive voice

File: doc/user/content/overview/api-components.md [hunk 4]
```diff
@@ -8,8 +8,7 @@ menu:
 ---
 
 Materialize is a streaming database with a SQL API. However, despite the fact
-that Materialize uses SQL idioms and can process data from databases, it
-actually has very little in common with "databases" as most users think of them.
+that Materialize uses SQL idioms and can process data from databases, it actually has very little in common with "databases" as most users think of them.
```
Comment (sploiselle) [inline doc/user/content/overview/api-components.md:11]: Just a heads up that these line breaks are intentional and should be maintained; all text should be wrapped at some character count. Most editors have some kind of plugin that will do this for you.

File: doc/user/content/overview/architecture.md [hunk 5]
```diff
@@ -8,8 +8,7 @@ menu:
 ---
 
 Everything in Materialize is currently handled by the `materialized` process
-(pronounced _materialize-dee_; the "`d`" is for daemon), which interacts with
-the outside world by interfacing with:
+(pronounced _materialize-dee_; the "`d`" is for daemon), which interacts with the outside world by interfacing with:
```
Comment (sploiselle) [inline doc/user/content/overview/architecture.md:11]: removed line wrap

File: doc/user/content/overview/volatility.md [hunk 6]
```diff
@@ -18,8 +18,7 @@ We call a source that cannot uphold this guarantee a *volatile* source. Many
 common sources of streaming data are volatile. For example, [Amazon
 Kinesis](https://aws.amazon.com/kinesis/) streams are volatile, because (by
 default) data is only retained for 24 hours. If Materialize restarts after
-reading a Kinesis stream for 25 hours, it will be unable to recover the first
-hour of data.
+reading a Kinesis stream for 25 hours, it will be unable to recover the first hour of data.
```
Comment (sploiselle) [inline doc/user/content/overview/volatility.md:21]: removed line wrap

File: doc/user/content/ops/deployment.md [hunk 7]
```diff
@@ -6,8 +6,7 @@ menu:
     parent: operations
 ---
 
-_This page is a work in progress and will have more detail in the coming months.
-If you have specific questions, feel free to [file a GitHub
+_This page is a work in progress and will have more detail in the coming months. If you have specific questions, feel free to [file a GitHub
```
Comment (sploiselle) [inline doc/user/content/ops/deployment.md:9]: Removed line wrap

**Review by sploiselle — COMMENTED** [review]
Realize that some of these changes likely belong elsewhere, but wanted to flag them while they were in front of me. The `show tables` stuff looks fine.

File: doc/user/content/overview/arangements.md (hunk 1 above)
Comment (msudberg) [reply doc/user/content/overview/arangements.md]: Yeah, sorry, that wasn't ready for review yet.

**Comment by msudberg** [comment]
Canceling because too many unwanted changes bundled in.

**Pull request closed without merging**